go/e2e/channel_test.go

```diff
@@ -5,12 +5,11 @@ import (
 	"crypto/x509"
 	"fabric-admin-sdk/pkg/channel"
 	"fmt"
-	"io/ioutil"
 	"os"
 
 	"github.com/hyperledger-twgc/tape/pkg/infra/basic"
-	"github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/ginkgo/v2"
+	ginkgo "github.com/onsi/ginkgo/v2"
```

Comment:
I'm not sure why ginkgo is imported twice. It seems like once should be enough

Reply:
I suppose due to we invoked ginkgo.skip instead of use skip() directly, going to fix it.